src/main.ts

```diff
@@ -605,12 +610,14 @@ export class Purchases {
         reject(new PurchasesError(ErrorCode.UserCancelledError));
       };
 
-      window.addEventListener("popstate", onClose);
+      if (!isInElement) {
```

Comment:
added this @nicfix , in case user has pushed state themselves, never want to listen to popstate either